routing/leaps_postprocessor.cpp

```diff
@@ -41,16 +41,69 @@ namespace routing
 // static
 size_t const LeapsPostProcessor::kMaxStep = 5;
 
-LeapsPostProcessor::LeapsPostProcessor(std::vector<Segment> const & path, IndexGraphStarter & starter)
-  : m_path(path),
-    m_starter(starter),
-    m_bfs(starter),
-    m_prefixSumETA(path.size(), 0.0)
+/// \brief It is assumed that there are no cycles in path.
```

Comment:
`...there is no cycle...` http://languagelearningbase.com/6844/when-must-to-use-there-is-and-there-not-there-are-there-are-not